dojo/tools/sonarqube_api/importer.py

```diff
@@ -143,6 +154,84 @@ def import_issues(self, test):
 
         return items
 
+    def import_hotspots(self, test):
+
+        items = list()
+
+        try:
+            client, config = self.prepare_client(test)
+
+            if config and config.sonarqube_project_key:  # https://github.com/DefectDojo/django-DefectDojo/pull/4676 cases no. 5 and 8
+                component = client.get_project(config.sonarqube_project_key)
+            else:  # https://github.com/DefectDojo/django-DefectDojo/pull/4676 cases no. 2, 4 and 7
+                component = client.find_project(test.engagement.product.name)
+
+            hotspots = client.find_hotspots(component['key'])
+            logging.info('Found {} hotspots for project {}'.format(len(hotspots), component["key"]))
+
+            for hotspot in hotspots:
+                status = hotspot['status']
+
+                if self.is_reviewed(status):
+                    continue
+
+                type = 'SECURITY_HOTSPOT'
+                if len(hotspot['message']) > 513:
+                    title = textwrap.shorten(text=hotspot['message'], width=507)
+                else:
+                    title = hotspot['message']
```

Comment:
You don't need this complex if/else.
`textwrap.shorten()` have this logic inside.
```suggestion
                title = textwrap.shorten(text=hotspot['message'], width=500)
```

Reply:
```bash
>>> shorten("1 3 5 7 9", 8)
'1 [...]'
>>> len(shorten("1 3 5 7 9", 8))
7
```

Reply:
Done